test/Syntax/Inputs/serialize_class_decl.json

```diff
@@ -0,0 +1,265 @@
+{
```

Comment:
I actually don’t think we need this test. I’d like to remove these JSON tests because they create a lot of churn in the test suite — I’m gonna replace the existing ones with FileCheck based tests that test more structural properties of the JSON.